.github/actions/docker-build/action.yaml

```diff
@@ -29,11 +33,14 @@ runs:
       run: |
         mkdir src
         vcs import src < autoware.repos
+        if [ -n "${{ inputs.additional-repos }}" ]; then
+          vcs import src < ${{ inputs.additional-repos }}
+        fi
       shell: bash
 
     - name: Cache ccache
       uses: actions/cache@v4
-      if: ${{ github.ref == 'refs/heads/main' }}
+      if: ${{ github.ref == 'refs/heads/main' && inputs.additional-repos == '' }}
```

Comment:
Instead of excluding the nightly from the cache, we could assign a different key for them.

Reply:
```
key: ccache-${{ inputs.platform }}-${{ hashFiles('src/**/*.cpp') }}
        restore-keys: |
          ccache-${{ inputs.platform }}-
```
to
```
key: ccache-nightly-${{ inputs.platform }}-${{ hashFiles('src/**/*.cpp') }}
        restore-keys: |
          ccache-nightly-${{ inputs.platform }}-
```

Reply:
But until we add this option, current workaround is alright :+1: